### ISSUE TITLE
otm down?

### ISSUE THREAD

**Issue by vicb** [der-stefan/OpenTopoMap#346]
It looks like https://opentopomap.org/ is down and no more tile served :(

OTM is nice and useful, anything I could do to help bringing it up again?

**Comment by der-stefan** [comment]
Yes, the main server had to be updated and is currently not in use. Therefore, I set up a small vserver that serves all tiles from zoom 0 to 14 in the meantime. At the moment, the server seems to be overloaded by the number of users.

I try my best. My time is limited. As I always communicate, the OTM stands on no solid fundaments and should never be used for critical applications.

**Comment by vicb** [comment]
Thanks for your quick reply @der-stefan.

Again let me know if there is anything I can do to help.

I use OTM as a layer for an app for paragliding pilots (flyxc.app).
